oasis/models/rules.py

```diff
@@ -65,7 +76,21 @@ def compute(self, metric, query_expr, config, rules):
                             "start to match with rule"
                             .format(log_prefix=self.log_prefix,
                                     metric=metric, value=features_value))
-                self.match_rules(metric, features_value, rules)
+                with self.lock:
+                    self.report.metrics_report[metric]["predict_count"] = \
+                        self.report.metrics_report[metric]["predict_count"] + 1
+
+                is_match, not_match_rule = self.match_rules(metric, features_value, rules)
```

Comment:
if match, where is the report

Reply:
`match` is normal, so we not need report.